packages/apps/tools/locales/en/common.json

```diff
@@ -63,6 +63,8 @@
   "Testnet": "Testnet",
   "Cross Chain Transfers": "Cross Chain Transfers",
   "Invalid length of request key": "Invalid length of request key",
+  "invalid-pub-key-length": "Invalid length of public key",
```

Comment:
it looks like the other translation keys are not using kebab-case. Maybe good to keep it consistent.